ivre/db/mongo.py

```diff
@@ -904,6 +905,31 @@ def searchfile(self, fname):
             output=fname,
         )
 
+    def searchsmbshares(self, access='', hidden=None):
+        """Filter SMB shares with anonymous `access` (default: either
```

Comment:
What's the purpose of the parameter `hidden` ? You docstring for `access` but not for it :)